INC CREATEv1

```diff
@@ -0,0 +1,197 @@
+from selenium import webdriver
+import time
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support.ui import Select
+from datetime import date
+from selenium.webdriver.common.action_chains import ActionChains
+import unittest
+
+class Servicenow_IncCreate(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Firefox()
+#        base_url =  "https://dev65721.service-now.com"
+        
+#        driver.implicitly_wait(10)
+
+#        
+
+#        driver.get(base_url)
+
+    def test_selenium(self):	
+        #
+        identifier = "AUTOMATED TESTING"
+        module = "INCIDENT"
+        action = "CREATE"
+        job_name = ""
+        today = str(date.today())
+        
+        driver = self.driver
+        
+        base_url =  "https://dev65721.service-now.com"
+        
+        driver.get(base_url)
+        
+        
+        driver.switch_to_frame("gsft_main")
+        username = driver.find_element_by_id("user_name")
+        username.send_keys("admin")
+
+        password = driver.find_element_by_id("user_password")
+        password.send_keys("Liverpool28")
+        password.send_keys(Keys.RETURN)
+
+        identifier_inc = ("AUTOMATED TESTING - INCIDENT - %s" %today)
+
+        # depreciated warning
+        driver.switch_to.default_content()
+        # driver.switch_to.frame
+
+        time.sleep (10)
```

Comment:
Do not use time.sleep where you can help it. Use an implicit wait or loop until element is available